From 944ae4d2df0f15afb29ac0a68d0dd012d9df9acf Mon Sep 17 00:00:00 2001
From: David Teigland <teigland@redhat.com>
Date: Wed, 15 Jun 2016 16:51:36 -0500
Subject: pvscan: fix lvmlockd global lock

This should have been removed when pvscan was
recently converted to use process_each_pv which
does the lvmlockd locking.
---
 tools/pvscan.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tools/pvscan.c b/tools/pvscan.c
index 5987e8b..78dad82 100644
--- a/tools/pvscan.c
+++ b/tools/pvscan.c
@@ -624,10 +624,6 @@ int pvscan(struct cmd_context *cmd, int argc, char **argv)
 		return ECMD_FAILED;
 	}
 
-	/* Needed for a current listing of the global VG namespace. */
-	if (!lockd_gl(cmd, "sh", 0))
-		return_ECMD_FAILED;
-
 	if (!(handle = init_processing_handle(cmd))) {
 		log_error("Failed to initialize processing handle.");
 		ret = ECMD_FAILED;
